src/wizard/dotnet/aspnet.md

```diff
@@ -51,6 +56,43 @@ public class MvcApplication : HttpApplication
         // Flushes out events before shutting down.
         _sentry?.Dispose();
     }
+    
+    protected void Application_BeginRequest()
+    {
+        // Start a transaction that encompasses the current request
+        var transaction = Context.StartSentryTransaction();
+
+        // Attach transaction to the request context to finish it when the request ends
+        Context.Items["__SentryTransaction"] = transaction;
+    }
+
+    protected void Application_EndRequest()
+    {
+        // Finish the currently active transaction
+        if (Context.Items.Contains("__SentryTransaction"))
+        {
+            var transaction = Context.Items["__SentryTransaction"] as ISpan;
+
+            var status =
+                Context.Response.StatusCode < 400 ? SpanStatus.Ok :
```

Comment:
We have to recall to change these 3 snippets once the extension method for closing is done